.pre-commit-config.yaml

```diff
@@ -1,14 +1,10 @@
 repos:
-    - repo: https://github.com/psf/black
-      rev: 23.3.0
-      hooks:
-          - id: black
-            exclude: _pb2\.pyi?$
     - repo: https://github.com/charliermarsh/ruff-pre-commit
-      rev: "v0.0.290"
+      rev: "v0.1.3"
```

Comment:
Is it possible to pin this more loosely? Like >=v0.1.3?

Reply:
it cannot and that's by design https://stackoverflow.com/a/75920492